packages/dart/npt_flutter/lib/localization/app_en.arb

```diff
@@ -100,17 +101,23 @@
    "onboardingSubTitle": "to NoPorts Desktop",
    "onboardingTitle": "Welcome",
    "overrideAllProfile": "Override all profiles with default relay selection",
+   "pasteJsonYaml": "Paste JSON/YAML",
```

Comment:
Not "JSON/YAML", non-developers have no idea what these file formats are, please use the word "Profile"

Reply:
Speaking of which, we should probably change the "YAML" export button to say "YAML (recommended)". I think YAML will be easier for most people, compared to JSON